portfolio/src/main/webapp/nav.js

```diff
@@ -0,0 +1,9 @@
+
+document.write('\
+  <div class="topnav">\
+    <a class="active" href="index.html">Home</a>\
+    <a href="hobbies.html">Hobbies</a>\
+    <a href="programming.html">Programming</a>\
+  </div> \
+  '
+);
```

Comment:
Is this file necessary? Would it be possible to insert this directly into the index.html page?

Reply:
The idea here was to be able to cleanly insert the nav bar into each individual page so I can edit the nav bar once for changes everywhere. Linh suggested the idea, but I'm not sure if I implemented it like she intended.

Reply:
Got it, this is good then. My only suggestion then would be to separate the raw string into a variable to reference inside of document.write()